docs/sources/reference/components/prometheus/prometheus.remote.queue.md

```diff
@@ -0,0 +1,296 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/prometheus/prometheus.remote.queue/
+description: Learn about prometheus.remote.queue
+title: prometheus.remote.queue
+---
+
+
+<span class="badge docs-labels__stage docs-labels__item">Experimental</span>
+
+# prometheus.remote.queue
```

Comment:
I know it may be late to the partey, but what do you think of `prometheus.write.queue`?
We have `loki.write`, `pyroscope.write`... to me it seems a good idea to use `prometheus.write.*` prefix and then we can have a `prometheus.write.sync` for maybe an implementation that writes synchronously without a WAL (can be useful for gateway scenario) and the current `prometheus.write.queue` that uses a queue....

Reply:
I dont have problem with that @ptodev @wildum thoughts? Seems to match better with our current standard so sounds legit.

Reply:
I like this better, and going to do it unless we get strong objection :)

Reply:
`prometheus.write.queue` sounds better to me too.

Reply:
Doing a rename here and will do the import changes in a separate PR I will automerge since it generates a lot of noise.